### DISCUSSION TITLE
Need help for DESIJ2318-1106

### DISCUSSION THREAD

**Discussion by rahulsust** [AstroBridge/BDLensing#131] [Q&A]
Here is my notebook https://github.com/rahulsust/BDLensing-Project/blob/main/lens_systems/DESIJ2318-1106/lens_model_DESIJ2318-1106.ipynb
I am confused if it is okay or not.

**Comment by mh-tanver** [comment]
1. Try masking out the shape at 8 o clock. Also you might consider the shapes at 2, 3, 4 o clock on the arc. 
2. update your **Visualizing the fitted model** cell adding `image_likelihood_mask_list=kwargs_likelihood["image_likelihood_mask_list"]` in the `model_plot = ModelPlot( )`.

**Comment by rafee-adnan** [comment]
The following suggestions may help with your fitting:
- Make the outer mask bigger, the arc is very close to the mask. Make the radius of the mask around 6.5”
- If you want to successfully fit the arc on the right-hand side, you have to make the `theta_E` much larger. As you can from your lens system, the value of the einstein radius is around 4.5 arc second. Set the lower limit high around 3.8”.
- From my observation, your `beta` value is very high. Try setting it low.
- Again, your `R_sersic` is higher than usual. Try constraining it to a lower value. You can try setting the upper limit to something like 0.2” for both `R_sersic` and `beta`.

**Reply by rahulsust** [reply]
Vai kindly check the current state of my notebook [here](https://colab.research.google.com/drive/1UG-_1Dc705rXTWEJJoD8MzLG5TDH4qQ7#scrollTo=257cc774)

**Reply by rafee-adnan** [reply]
The link does not work. It requires access.

**Reply by rahulsust** [reply]
Sorry about that vai. Here is the open [link](https://colab.research.google.com/drive/1UG-_1Dc705rXTWEJJoD8MzLG5TDH4qQ7?usp=sharing)

**Reply by rafee-adnan** [reply]
- Try to follow the suggestions for #130, as you are having similar problem. Instead of getting greater gamma value, your gamma value is quite low. 
- Additionally, inside `model_plot.source_plot()` add these two `with_caustics=True, scale_size=0.5`. Adjust the scale size according to the size of your reconstructed source.
